### PR TITLE
tlshelpers: add package

### PR DESCRIPTION
**Pull request by D3vil0p3r**

### CODE REVIEW

File: packages/tlshelpers/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# See COPYING for license details.
+
+pkgname=tlshelpers
+pkgver=21.6e422be
+pkgrel=1
+pkgdesc='A collection of shell scripts that help handling X.509 certificate and TLS issues.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-recon')
+url='https://github.com/hannob/tlshelpers'
+license=('custom:UNLICENSE')
```
Comment (noraj) [inline packages/tlshelpers/PKGBUILD:11]:
```suggestion
license=('Unlicense')
```

cf. `/usr/share/licenses/common/Unlicense/`

File: packages/tlshelpers/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+package() {
+  cd $pkgname
+
+  install -dm 755 "$pkgdir/usr/bin"
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" *.md
+
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+
+  rm -rf LICENSE *.md .gitignore
```
Comment (noraj) [inline packages/tlshelpers/PKGBUILD:31]: not useful if not doing a `cp *` afterward

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/tlshelpers/PKGBUILD (hunk 2 above)
Comment (D3vil0p3r) [reply packages/tlshelpers/PKGBUILD]: > not useful if not doing a `cp *` afterward

Right, I deleted `rm -rf LICENSE *.md .gitignore`. Should I delete also `install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"`?

Comment (noraj) [reply packages/tlshelpers/PKGBUILD]: Yes as it's a common license.

Comment (D3vil0p3r) [reply packages/tlshelpers/PKGBUILD]: > Yes as it's a common license.

Deleted as requested.

**Pull request merged**